src/js/actions/history.js

```diff
@@ -77,51 +77,54 @@ define(function (require, exports) {
     queryCurrentHistory.modal = true;
 
     /**
-     * Given a history state event from photoshop, dispatch a flux event for the history store
+     * Helper function to load a state from history store based on a count offset
+     * and then to fix it up with the latest selection state, visibility state, and
+     * then reset border policies
      *
-     * @param {object} event a raw historyState event from photoshop
+     * @private
+     * @param {number} documentID
+     * @param {number} count
      * @return {Promise}
      */
-    var handleHistoryState = function (event) {
```

Comment:
This action was mostly just moved down in the file, but the "id" was also added to the payload.